src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py

```diff
@@ -112,6 +112,18 @@ def get_vm_size_completion_list(prefix, action, parsed_args, **kwargs):#pylint:
 register_cli_argument('vm extension image', 'type', CliArgumentType(options_list=('--name', '-n')))
 register_cli_argument('vm extension image', 'latest', CliArgumentType(action='store_true'))
 
+register_cli_argument('vmss extension', 'extension_name', name_arg_type, id_part='name', help='Name of extension')
+register_cli_argument('vmss extension', 'vmss_name', help='Scale set name')
+register_cli_argument('vmss extension', 'auto_upgrade_minor_version', CliArgumentType(action='store_true'))
```

Comment:
If we are adding new code, we should not copy-paste bad patterns. Even if we don't update VM extensions in this round, we shouldn't knowingly perpetuate a bad practice.

---
In reply to: [76100584](https://github.com/Azure/azure-cli/pull/737#discussion_r76100584) [](ancestors = 76100584,76077802)